src/test/java/io/confluent/connect/jdbc/source/JdbcSourceTaskUpdateTest.java

```diff
@@ -316,11 +369,39 @@ public void testManualIncrementingRestoreOffset() throws Exception {
     PowerMock.verifyAll();
   }
 
+  @Test
+  public void testAutoincrementRestoreNoVersionOffset() throws Exception {
+    TimestampIncrementingOffset offset = new TimestampIncrementingOffset(null, 1L);
+    testAutoincrementRestoreOffset(
+        Collections.singletonMap(SINGLE_TABLE_PARTITION, offset.toMap())
+    );
+  }
+
+  @Test
+  public void testAutoincrementRestoreVersionOneOffset() throws Exception {
+    TimestampIncrementingOffset offset = new TimestampIncrementingOffset(null, 1L);
+    testAutoincrementRestoreOffset(
+        Collections.singletonMap(SINGLE_TABLE_PARTITION_WITH_VERSION, offset.toMap())
+    );
+  }
+
   @Test
   public void testAutoincrementRestoreOffset() throws Exception {
```

Comment:
This is testing when we have offsets with both protocols. Is that worth having the method include that information? E.g., `testAutoincrementRestoreOffsetsWithMultipleProtocol()`